Sources/Client/Views/Styles/Text/WarningTextStyle.swift

```diff
@@ -0,0 +1,24 @@
+import SwiftUI
+
+
+// MARK: - WarningTextStyle
+
+
+struct WarningTextStyle: TextStyle {
+  
+  // MARK: - Properties.UI
+  
+  
+  private let warningTextColor = Colors.warningYellow.color
+  
```

Comment:
there are a lot of double empty lines, please go through and remove them all, it doesn't follow the code style i've chosen for this project.